posthog/clickhouse/schema.py

```diff
@@ -63,6 +64,17 @@
     APP_METRICS_MV_TABLE_SQL,
 )
 
+CREATE_KAFKA_DLQ_QUERIES = (
```

Comment:
Tests won't capture the schema of this until you also update `CREATE_TABLE_QUERIES` below.

Would be great to see the snapshot diff as well for the review - mind running bin/tests manually on this file?